aztec/src/main/kotlin/org/wordpress/aztec/spans/AztecDynamicImageSpan.kt

```diff
@@ -109,6 +116,14 @@ abstract class AztecDynamicImageSpan(val context: Context, protected var imageDr
             height = (width / aspectRatio).toInt()
         }
 
+        // Note: This is not a solution just a temp code
+        // to demonstrate that for some reason value 36 ( which is got
+        // from imageDrawable?.intrinsicHeight ) is causing overlap problem
+        // or I think it's causing :D
+        if (height == 36) {
```

Comment:
This is the magic number from the logs. I am not sure why `imageDrawable?.intrinsicHeight` at some point is returning this value and also not sure if it's the case only for emulator or general.